assets/scripts/app/__tests__/__snapshots__/NoConnectionMessage.test.js.snap

```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Below, are the two new snapshots we created in our updated tests above. New snapshots or changes to existing snapshots should be committed alongside the test!